suc/pkg/service/config.go

```diff
@@ -0,0 +1,99 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rancher/wins/cmd/server/config"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ConfigDirEnvVar      = "CATTLE_WINS_CONFIG_DIR"
+	DebugEnvVar          = "CATTLE_WINS_DEBUG"
+	AgentStringTLSEnvVar = "STRICT_VERIFY"
+)
+
+const (
+	defaultConfigFile = "c:/etc/rancher/wins/config"
+)
+
+func getConfigPath(path string) string {
+	configDirEnv := os.Getenv(ConfigDirEnvVar)
+	if path == "" && configDirEnv != "" {
+		path = configDirEnv
+	}
+	if path == "" {
+		path = defaultConfigFile
+	}
+	return path
+}
```

Comment:
```suggestion
func getConfigPath(path string) string {
  if path != "" {
    return path
  }
	if path = os.Getenv(ConfigDirEnvVar); path != "" {
	  return path
	}
	return defaultConfigFile
}
```
The first branch is somewhat confusing, but I've deduced the following:
1. If the path parameter is not empty, return that,
2. otherwise, return the value of the env var
3. otherwise, return the default

Reply:
It also looks like `UpdateConfigFromEnvVars` is only ever called with `""`, so do we really need the `path` parameter?

Reply:
Dropped the `path` parameter